packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -379,14 +379,14 @@ const RealAccountSignup = ({
             localStorage.removeItem('real_account_signup_wizard');
         }
 
-        if (modal_content[getActiveModalIndex()].action !== 'signup') {
-            closeRealAccountSignup();
+        if (modal_content[getActiveModalIndex()].action === 'signup' && e) {
```

Comment:
What is `&& e` for?

Reply:
I've checked for when user click on the 'x' button it will give values stating that the target is `svg.close` icon while if user clicked on `maybe later` the value `e` will be undefined. I'm checking whether e is undefined so that it won't render the pop up when user click on `Maybe Later` button. It will only show up when user is on signup and e has a value

Reply:
@suisin-deriv can we add a comment in the code to avoid confusion.